tsl/src/hypercore/hypercore_handler.c

```diff
@@ -3886,7 +3896,7 @@ convert_index_only_scans(const HypercoreInfo *hsinfo, List *pathlist)
 			for (int i = 0; i < indkeys->dim1; i++)
 			{
 				const AttrNumber attno = indkeys->values[i];
-
+				const HypercoreInfo *hsinfo = RelationGetHypercoreInfo(rel);
```

Comment:
```suggestion
				const HypercoreInfo *hcinfo = RelationGetHypercoreInfo(rel);
```

Might as well update the name while we are at it.